vms/platformvm/service.go

```diff
@@ -511,6 +511,83 @@ func (s *Service) GetUTXOs(_ *http.Request, args *api.GetUTXOsArgs, response *ap
 	return nil
 }
 
+/*
+ *****************************************************
+ ******************* Get Subnet **********************
+ *****************************************************
+ */
+
+// GetSubnetArgs are the arguments to GetSubnet
+type GetSubnetArgs struct {
+	// ID of the subnet to retrieve information about
+	SubnetID ids.ID `json:"subnetID"`
+}
+
+// GetSubnetResponse is the response from calling GetSubnet
+type GetSubnetResponse struct {
+	// ID of the subnet
+	SubnetID ids.ID `json:"subnetID"`
+	// whether it is permissioned or not
+	IsPermissioned bool `json:"isPermissioned"`
+	// subnet auth information for a permissioned subnet
+	ControlKeys []string    `json:"controlKeys"`
+	Threshold   json.Uint32 `json:"threshold"`
```

Comment:
I think we would need to include the locktime here as well

Reply:
what is it used for?

Reply:
https://github.com/ava-labs/avalanchego/blob/master/vms/secp256k1fx/fx.go#L183-L184